app/src/main/res/menu/queue.xml

```diff
@@ -28,6 +28,11 @@
         android:title="@string/sort">
 
         <menu>
+            <item
+                android:id="@+id/queue_sort_automatically"
+                android:title="@string/sort_automatically"
+                android:checkable="true" />
+
```

Comment:
I think this should rather be the last item of the sort menu

Reply:
I will move down the item.